### PR TITLE
Added linkes to all RFCs

### PR DESCRIPTION
**Pull request by pvlakshm**
Added links to all RFCs to address #1392.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,21 @@ There are many ways to contribute to VSTest
 - [Data collection Protocol](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0006-DataCollection-Protocol.md)
 - [Translation Layer](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0008-TranslationLayer.md)
 - [Editors API Revision Update](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0009-Editors-API-RevisionUpdate.md)
+- [TranslationLayer](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0008-TranslationLayer.md)
+- [Source-Information-For-Discovered-Tests](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0010-Source-Information-For-Discovered-Tests.md)
```
Comment (singhsarab) [inline README.md]: We can avoid "-" just for better readability.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,21 @@ There are many ways to contribute to VSTest
...
+- [Test-Adapter-Lookup](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0013-Test-Adapter-Lookup.md)
+- [Packaging](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0014-Packaging.md)
+- [Telemetry](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0015-Telemetry.md)
+- [Loggers-Information-From-RunSettings](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0016-Loggers-Information-From-RunSettings.md)
+- [TestCase-FullyQualifiedName](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0017-TestCase-FullyQualifiedName.md)
+- [Skip-Default-Adapters](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0018-Skip-Default-Adapters.md)
+- [Disable-Appdomain-While-Running-Tests](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0019-Disable-Appdomain-While-Running-Tests.md)
+- [Improving-Logic-To-Pass-Sources-To-Adapters](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0020-Improving-Logic-To-Pass-Sources-To-Adapters.md)
+- [CodeCoverageForNetCore](https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0021-CodeCoverageForNetCore.md)
```
Comment (singhsarab) [inline README.md]: Nit: Expand the Title for this one.

File: README.md (hunk 1 above)
Comment (pvlakshm) [reply README.md]: "-" is for getting it rendered as a list.

Comment (singhsarab) [reply README.md]: In the name pratap sir.

**Review by singhsarab — APPROVED** [review]

**Comment by mmitche** [comment]
@dotnet-bot test this please

**Comment by singhsarab** [comment]
@dotnet-bot test Ubuntu14.04 / Release Build

**Comment by singhsarab** [comment]
@dotnet-bot test Ubuntu14.04 / Debug Build

**Pull request merged**